src/actions/hotel.js

```diff
@@ -1,15 +1,19 @@
 import { VIEW_HOTEL, SET_HOTEL_PAGE } from '../constants/actionTypes';
-import { QUERY_FETCH_BUCKET_ITEM } from '../constants/queries';
-import * as graphqlService from '../services/graphql';
 
 export const viewHotel = () => { return {type: VIEW_HOTEL}; };
 
 export const setHotelPage = (hotel) => { return {type: SET_HOTEL_PAGE, hotel}; };
 
 export const getHotel = (bucketId, itemId) => {
   return (dispatch) => {
-    return graphqlService.query(QUERY_FETCH_BUCKET_ITEM, {'id': bucketId, 'itemId': itemId, 'itemType': 'package'})
-      .then((data) => { console.log(data); dispatch(setHotelPage(data.data.viewer.searchItem.packageOffer)); })
-      .catch((err) => console.log(err));
+    return new Promise((resolve, reject) => {
```

Comment:
⚠️  Careful... unless we are *explicitly* bundling a `Promise` library in the client, we should not rely on browsers to have a Native implementation... None of the internet explorer versions have promises. Which means that Rasmus, Roy etc will not have it...

see: http://caniuse.com/#feat=promises
![promises-not-available-in-internet-explorer](https://cloud.githubusercontent.com/assets/194400/15707736/3b5ac53e-27f2-11e6-983e-e813cc030916.png)